weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -662,6 +664,74 @@ export const CallsTable: FC<{
     setSelectedCalls([]);
   }, [setSelectedCalls]);
 
+  // Detect peek drawer state to flip proportions
```

Comment:
These hooks pair with 

```
<div
            className={`resize-handle ${isDragging ? 'dragging' : ''}`}
            onMouseDown={handleMouseDown}
          />
```

I would challenge you to make a generic `Draggable` esk component that hides these implementation details. Then the CallsTable just includes that with some params